articles/automation-testing-in-selenium/index.md

```diff
@@ -0,0 +1,99 @@
+Digital revolution has led to an increase in the number of web applications. Developers are constantly creating new web applications to meet various demands. Before these applications are commissioned for use, developers test them (using automation testing tools) to ensure that they function without problems. Selenium is an effective automation tool that can help developers to validate their applications. 
+
+This article will take you through fundamental aspects of this software. It provides an overview of how automation testing takes place in Selenium software. The article has also highlighted the main types of testing done by Selenium.  
+
+### Introduction to Selenium Automation Testing
+Selenium is a software used for automation testing in new applications. This software provides a framework for the testing of web applications (across various web platforms). It is open-source, which means that developers will not incur costs related to licensing. 

```

Comment:
> Selenium is a software used for automation testing in new applications.

Please give a brief description of what Automation Testing is.

Reply:
Done.